### PR TITLE
CI: Disable certain labels from becoming stale

### PR DESCRIPTION
**Pull request by Pandapip1**
These labels mean that the PR is waiting on some other person in order to be merged, and the author may still be responsive.

### CODE REVIEW

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -24,11 +24,12 @@ jobs:
         close-issue-message: This issue was closed due to inactivity. If you are still pursuing it, feel free to reopen it and respond to any feedback.
         days-before-issue-stale: 7
         days-before-issue-close: 49 # 49 + 7 weeks = 3 months
-        exempt-issue-labels: discussions-to
+        exempt-issue-labels: discussions-to, e-consensus
```
Comment (xinbenlv) [inline .github/workflows/stale.yml:27]: What is "e-consensus"? just curious?

Comment (Pandapip1) [reply .github/workflows/stale.yml:27]: Waiting on editor consensus

**Comment by Pandapip1** [comment]
True... I'd still argue e-consensus and e-review should be exempt.

**Review by SamWilsn — APPROVED** [review]

**Comment by Pandapip1** [comment]
Codespell is failing. Manual merge neeed.

**Comment by Pandapip1** [comment]
Manually reran CI

**Pull request merged**